content/blog/2020-07-07-cml-release.md

```diff
@@ -0,0 +1,182 @@
+machine learning projects:
+
+1. **Data dependencies.** In ML, data plays a similar role as code: ML results
+   critically depend on datasets, and changes in data need to trigger feedback
+   just like changes in source code. Furthermore, multi-GB datasets are
+   challenging to manage with Git-centric CI systems.
+2. **Metrics-driven.** The traditional software engineering idea of pass/fail
+   tests does not apply in ML. As an example, `+0.72% accuracy` and
+   `-0.35% precision` does not answer the question if the ML model is good or
+   not. Detailed reports with metrics and plots are needed to make a good/bad
+   model discussion
+3. **CPU/GPU resources**. ML training often requires more resources to train
+   then is typical to have in CI/CD runners. CI/CD must be connected with cloud
+   computing instances or Kubernetes clusters for ML training.
+
+## 2. CI/CD for ML is the next step for DVC team
+
+Since the beginning, our motivation has been helping ML teams benefit from
+DevOps. We started DVC because we knew that data management would be a crucial
+bottleneck, and sure enough, DVC was a big step towards making pipelines and
+experiments manageable and reproducible. But conversations with our community
+have brought us to one conclusion again and again: CI/CD for ML is the holy
+grail.
+
+Over the last 3 years, we've reached some big milestones:
+
+1. We built DVC to address the ML data management problem. Recently, we
+   [released DVC 1.0](https://dvc.org/blog/dvc-1-0-release), marking a new and
+   more stable era for our API.
+2. DVC has become a core part of many ML team's daily operations. The latest
+   [ThoughtWorks Technology Radar](https://www.thoughtworks.com/radar/tools)
+   says:
+
+   _"... it [DVC] has become a favorite tool for managing experiments in machine
+   learning (ML) projects. Since it's based on Git, DVC is a familiar
+   environment for software developers to bring their engineering practices to
+   ML practice."_
+
+3. An extraordinary team and community have emerged around DVC:
+   - 15 employees in our organization https://iterative.ai
+   - 100+ open-source contributors to DVC https://github.com/iterative/dvc and
+     another 100+ open-source contributors to docs
+     https://github.com/iterative/dvc.org
+   - 2000+ community members in our Discord https://dvc.org/chat and GitHub
+     issue tracker https://github.com/iterative/dvc
+   - 4000+ regular users of DVC
+
+Now that DVC is maturing, we're ready to take the next step: we want to
+revolutionize ML development processes. We want ML experiments to have greater
+visibility to teammates, shorter feedback loops, and more reproducibility. We
+want teams to spend less time managing their computing resources and
+experiments, and more time building value. The goal is to extend the amazing
+results of DevOps from software development to ML and MLOps.
+
+## 3. Continuous Machine Learning release
```

Comment:
I'd use italics here because on first glance it reads like "a continuous release of machine learning"
```suggestion
## 3. _Continuous Machine Learning_ release
```
Or add "(CML)" before "release".

Reply:
Or
```suggestion
## 3. Continuous Machine Learning (CML) release
```